source/common/common/utility.cc

```diff
@@ -122,6 +122,15 @@ std::vector<std::string> StringUtil::split(const std::string& source, const std:
   return ret;
 }
 
+std::string StringUtil::join(const std::vector<std::string>& source, const std::string& delimiter) {
```

Comment:
Thanks, fixed that up in https://github.com/lyft/envoy/pull/1018/commits/9d919fd22b2118b9b54909f16278e29b2c35d87a.